packages/lending/src/protocols/compound-v3/lending-protocol.ts

```diff
@@ -365,6 +365,24 @@ export class LendingProtocol extends Protocol {
     return Promise.all(configMap[this.chainId].markets.map(({ id }) => this.getPortfolio(account, id)));
   }
 
+  async getProtocolInfo(marketId: string) {
+    const { baseToken, assets } = await this.getMarket(marketId);
+
+    return {
+      chainId: this.chainId,
+      protocolId: this.id,
+      marketId,
+      reserveTokens: [
+        { isSupplyEnabled: true, isBorrowEnabled: false, asset: baseToken },
+        ...assets.map(({ token }) => ({ isSupplyEnabled: false, isBorrowEnabled: true, asset: token })),
```

Comment:
does assets mean collaterals? if yes, I think collaterals are only suppliable not borrowable